idutils/__init__.py

```diff
@@ -384,6 +384,10 @@
 )
 """See https://www.wikidata.org/wiki/Property:P214."""
 
+email_regexp = re.compile(r"\S+@(\S+\.)+\S+")
+
+sha1_regexp = re.compile(r"^[a-zA-Z0-9]{40}$")
```

Comment:
```suggestion
sha1_regexp = re.compile(r"^[a-fA-F0-9]{40}$")
```

Only hexadecimal right?